packages/apps/composer-app/package.json

```diff
@@ -10,6 +10,7 @@
   "author": "DXOS.org",
   "scripts": {
     "build:functions": "esbuild ./src/functions/_worker.ts --bundle --format=esm --outfile=./out/composer/_worker.js",
+    "copy:assets": "mkdir -p ./public/assets/plugin-sketch && cp -R node_modules/@braneframe/plugin-sketch/dist/assets/{embed-icons,fonts,icons,translations} ./public/assets/plugin-sketch",
```

Comment:
Seems like something we should defer to Vite since it’s built for just this.

https://vitejs.dev/guide/assets

Reply:
As long as TLDraw is configured to defer to Vite bundling instead of using CDN assets, assets should land in the bundle just fine.

Reply:
TLDraw has docs on this: https://tldraw.dev/installation#Using-a-bundler

Reply:
@thure I think that @richburdon tried that and it didn't work. My guess would be because of the intermediate plugin compilation step. We'll likely be able to resolve this when we start bundling plugins independently and loading them at runtime.

Reply:
I see, okay!